src/Microsoft.Graph/Generated/Groups/Item/Conversations/ConversationsRequestBuilder.cs

```diff
@@ -62,8 +62,8 @@ public class ConversationsRequestBuilder : BaseRequestBuilder {
             return await RequestAdapter.SendAsync<ConversationCollectionResponse>(requestInfo, ConversationCollectionResponse.CreateFromDiscriminatorValue, errorMapping, cancellationToken);
         }
         /// <summary>
-        /// Use reply thread or reply post to further post to that conversation.
-        /// Find more info here <see href="https://learn.microsoft.com/graph/api/group-post-conversations?view=graph-rest-1.0" />
+        /// Create an open extension (openTypeExtension object) and add custom properties in a new or existing instance of a resource. You can create an open extension in a resource instance and store custom data to it all in the same operation, except for specific resources. The table in the Permissions section lists the resources that support open extensions.
```

Comment:
The description is confusing for the method that creates a new conversation. The same description is already there for `ExtensionsRequestBuilder`s.